forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+  remain anonymous.
+
+  TODO Do we want to alter the text at all?
+\end{resp}
+
+
+
+  5. What are Change Healthcare claims..as opposed to others?
+
+
+\begin{resp}
+  Hopefully the discussion to point 4 above has cleared up any confusion.
+\end{resp}
+
+
+\section*{Reviewer 2}
+\begin{quote}
+  \emph{(items numbered by the editor for convenience of reference)}
+\end{quote}
+
+\subsection*{Statistical or Methodological Comments:}
+  
+1. Is this really a stationary phenomenon? Over the timerange extending from
+pre-lockdown to today, it certainly isn't. The method in this paper of
+presenting model prediction errors makes it impossible to see this. People in
+econometrics look at cumsums of normalized prediction errors. You can see change
+points and trends very easily.
+
+\begin{resp}
+  There is little reason to believe that this data is stationary. The models
+  however are trained and evaluated over reasonably short periods (2-3 months),
+  over which time the behavior may well be stationary. We have added a section
+  to the appendix that uses the cumulative mean as suggested (the sum is
+  difficult to see due to the scaling). With some exceptions at the beginning of
+  the evaluation period, the relative ordering remains fairly consistent. The AR
+  model is easily the worst.
+
+  TODO I have added the code and a caption for the figure to the Supplement, but
+  no corresponding 
+  text. It seems a fairly minor addition, not necessarily deserving its own
+  section. 
+\end{resp}
+
+2. Are the results in this paper statistically significant? I see no discussion
+of this question, which seems unbelievable for me to be saying, given the
+authorship. And yet here I am.
+
+\begin{resp}
+  This is an important point that we should have addressed more fully.
+  Conventional methods for assessing statistical significance that we explored
+  were overconfident, to the point of being unbelievable. To a large degree,
+  this is a ``huge n, tiny p'' problem so even bootstrap analyses result in
+  extremely narrow intervals. Essentially, ``everything is significant''.
+
+  We also have strong reasons for not using model-based approaches (one being the
+  nonstationarity the Reviewer mentions above, but also the belief that these
+  models are wrong). We felt that the related investigation in 2B was more
+  pertinent.
+
+  To respond more directly:
+
+  $\bullet$ We have added a section to the Supplement (and point
+  to it around 
+  line 85 in the main text), that performs a Sign Test for differences in
+  accuracy. P-values there tend to cluster near zero suggesting again that these
+  improvements are statistically significant.
+
+  $\bullet$ We also use a Diebold-Mariano test
+  for comparing forecast accuracy. The results for this test are more mixed,
+  likely due to nonstationarity the Reviewer mentions above (the indicators are
+  more helpful in some periods than others, see also the upswing/downswing
```

Comment:
Is this referring to Section 2C?  If so, it'd be good to specify that for clarity.